src/test_parenthetics.py

```diff
@@ -0,0 +1,21 @@
+"""This tests the parenthetics module."""
+import pytest
+
+PAREN_TABLE = [
+    [-1, "())(()"],
+    [0, "()()()"],
```

Comment:
I would've liked to see more tests of balanced strings, like with nested parentheses for example.